task01-html-css/characters.css

```diff
@@ -0,0 +1,71 @@
+/**
+ * All Rights Reserved
+ * This software is property information of AKUREY S.A.
+ * Contact us at contact@akurey.com
+ * @summary short description for the file
+ */
+
+.grid-container {
+    display: flex;
+    flex-direction: row;
+    align-items: flex-start;
+    gap: 20px;
+
+    width: 940px;
+    height: 279.65px;
+    overflow-x: scroll;
+
+    flex: none;
+    order: 1;
+    flex-grow: 0;
```

Comment:
You can omit these lines, for now, not every single line of CSS code from Figma is necessary, we try to keep the code as short and effective as possible.